diff --git a/tensorflow/lite/BUILD b/tensorflow/lite/BUILD
index 597f81194cd..788decaec74 100644
--- a/tensorflow/lite/BUILD
+++ b/tensorflow/lite/BUILD
@@ -775,6 +775,8 @@ tflite_cc_shared_object(
         ":tflite_exported_symbols.lds",
         ":tflite_version_script.lds",
         "//tensorflow/lite/kernels:builtin_ops_all_linked",
+        "//tensorflow/lite/delegates/flex:delegate",
+        "//tensorflow/lite/delegates/xnnpack:xnnpack_delegate",
     ],
 )
 
